### ISSUE TITLE
TelegramLongPollingBot marked as deprecated but no hints on how to solve the problem

### ISSUE THREAD

**Issue by Polve** [rubenlagus/TelegramBots#1202]
Following the "Getting started" guide to create a base bot I extend the class TelegramLongPollingBot but while compiling I get this deprecation error:

> warning: [deprecation] TelegramLongPollingBot() in TelegramLongPollingBot has been deprecated

The problem is that both in the guide and in the API there is no hint on the way forward and what is the "right" solution to avoid the deprecation.

**Comment by Chase22** [comment]
It's not the class but the constructor that is deprecated. The issues with documentation are known and  i fixed them in #1194

You can find the updated GettingStarted here: https://github.com/Chase22/TelegramBots/blob/720f2767f5df101bbbb0bb868b404a579e495708/TelegramBots.wiki/Getting-Started.md#building-your-first-bot

Tldr: You need to pass your bot token into the super call in your bot classes constructor

**Comment by Polve** [comment]
Thank you for the details
